### ISSUE TITLE
Adding i18n support

### ISSUE THREAD

**Issue by r33beers** [rubyforgood/growhaus_sign_in#3]
Part of fixing this issue is providing instructions to the team members on how to define text in the views, so it is dynamic.

**Comment by r33beers** [comment]
We can set the locale by ```locale = :en``` or ```locale = :es```.

We get messages from the config files by using ```t 'some.key.name'```.

Messages are set in ```config/locales/*.yml```.

Example config file setup we can use:
```yaml
en:
  pagename:
    sectionofthepage:
      messagetitle: "Hello world"
```

**Comment by r33beers** [comment]
Closed by #19
